docs/proposals/formatted-output.md

```diff
@@ -272,51 +267,57 @@ localhost:localhost:5000/hello@sha256:5cb894d0c94c56894e160ad2eeb19a123b4d215537
     └── sha256:476d43120d3799fa76d7706f741beca73f5ff4149c8b6db3bd516a73d4a82fc1
 ```
 
-View an artifact's referrers manifest in pretty JSON output. 
+View an artifact's referrers manifest in pretty JSON output. The following fields should be outputted:
+
+- `OCI-Filters-Applied`: when the registry supports filtering on `artifactType`, this filed could be displayed
+- `Manifests`: the list of referrers
+  - `Ref`: full reference by digest of the referrer
+  - `MediaType`: media type of the referrer
+  - `Size`: referrer file size in bytes
+  - `Digest`: digest of the referrer
+  - `ArtifactType`: the type of a referrer
+  - `Annotations`: contains arbitrary metadata for the image manifest
```

Comment:
> [!NOTE]
>The annotations here is not the annotations for the image manifest but the annotations inside the image manifest.

Reply:
@shizhMSFT How about:
`Annotations`: contains arbitrary metadata of a referrer

Reply:
contains arbitrary metadata *in* a referrer?

Reply:
Sounds good to me. Updated.